hdwallet/src/main/kotlin/io/provenance/hdwallet/wallet/Wallet.kt

```diff
@@ -6,35 +6,40 @@ import io.provenance.hdwallet.bip39.DeterministicSeed
 import io.provenance.hdwallet.bip39.MnemonicWords
 import io.provenance.hdwallet.bip44.PathElement
 import io.provenance.hdwallet.bip44.PathElements
+import io.provenance.hdwallet.common.hashing.sha256
+import io.provenance.hdwallet.ec.CURVE
+import io.provenance.hdwallet.ec.Curve
 import io.provenance.hdwallet.ec.ECKeyPair
 import io.provenance.hdwallet.encoding.base58.base58DecodeChecked
 import java.security.KeyException
 
+/**
+ * Wallets are the root key representation used to derive [Account]s.
```

Comment:
👍  I really like the combination of KDoc and dokka for documentation. If devops adopts Nexus pro, you can view the HTML of javadoc jars directly from within Nexus. Imagine being able to link to the Javadoc from the repo README!

Reply:
That would be really useful! This one goes up to maven central though, not nexus.I haven't dug too much into their javadoc jar support yet.